phpstan.neon

```diff
@@ -10,7 +10,7 @@ includes:
 services:
     # require constant in argument position
     -
-        class: Symplify\CodingStandard\Rules\ForceMethodCallArgumentConstantRule
+        class: Symplify\CodingStandard\Rules\RequireMethodCallArgumentConstantRule
```

Comment:
Sneaky @TomasVotruba :D

Reply:
This should be fixed in `master`. Have you rebased? :)